src/inventory_ui.cpp

```diff
@@ -295,13 +337,38 @@ void inventory_selector::print_middle_column() const
     print_column(worn, middle_column_offset, middle_column_width, selected_w, current_page_offset_w);
 }
 
-void inventory_selector::print_left_column() const
+void inventory_selector::print_left_column(const bool show_direction) const
 {
-    print_column(items, left_column_offset, left_column_width, selected_i, current_page_offset_i);
+    print_column(items, left_column_offset, left_column_width, selected_i, current_page_offset_i, show_direction);
+}
+
+std::string suffix_direction ( int direction ) {
```

Comment:
At first, I was thinking of using advanced inventory drops. Did not do a full study yet but believe able if use advanced_inv_area (or something like that) and copy how advanced inventory drop the item.

Subsequently, discovered that the functions that drop calls already support dropping in other location.

Thanks, will look in direction